### PR TITLE
Introduce Support for Google Wahoo Platform

### PR DESCRIPTION
**Pull request by sc-ah**
### Description

This PR will add support for Pixel 2 and Pixel 2XL series

### Checklist

* [ y] Have you tested the change you are submitting?

Please keep in Mind Windows cant be booted this is only for Linux and Android multiboot / maintainence purposes
* [ y] Is the commits history nice and clean?

### CODE REVIEW

File: Platform/Google/msm8998/AcpiTables/walleye/dsdt_common.asl [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+{
+    Name (_HID, "QCOM0009")  // _HID: Hardware ID
+    Name (_UID, Zero)  // _UID: Unique ID
+    Alias (\_SB.PSUB, _SUB)
+}
+
+//
+// Storage - UFS/SD 
+//
+Include("ufs.asl")
```
Comment (qaz6750) [inline Platform/Google/msm8998/AcpiTables/walleye/dsdt_common.asl:36]: * Does not disable ufs cause problems?

File: Platform/Google/msm8998/walleye.dsc [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  SUPPORTED_ARCHITECTURES        = AARCH64
+  BUILD_TARGETS                  = DEBUG|RELEASE
+  SKUID_IDENTIFIER               = DEFAULT
+  FLASH_DEFINITION               = Platform/Qualcomm/msm8998/msm8998.fdf
+  DEVICE_DXE_FV_COMPONENTS       = Platform/Google/msm8998/walleye.fdf.inc
+
+!include Platform/Qualcomm/msm8998/msm8998.dsc
+
+[BuildOptions.common]
+  GCC:*_*_AARCH64_CC_FLAGS = -DENABLE_SIMPLE_INIT -DENABLE_LINUX_SIMPLE_MASS_STORAGE -DAB_SLOTS_SUPPORT
```
Comment (qaz6750) [inline Platform/Google/msm8998/walleye.dsc:17]: Should "AB_SLOTS_SUPPORT=TURE" not be used?

Comment (sc-ah) [reply Platform/Google/msm8998/walleye.dsc:17]: I enabled AB Slots in walleye platform

Comment (sc-ah) [reply Platform/Google/msm8998/walleye.dsc:17]: Is there a better way to enable AB slots support ??

File: Platform/Google/msm8998/AcpiTables/walleye/dsdt_common.asl (hunk 1 above)
Comment (sc-ah) [reply Platform/Google/msm8998/AcpiTables/walleye/dsdt_common.asl:36]: I never tried booting windows tbh i will disable it and push again

File: Platform/Google/msm8998/walleye.dsc (hunk 2 above)
Comment (qaz6750) [reply Platform/Google/msm8998/walleye.dsc:17]: yes，This improvement was added in the commit long ago

**Review by qaz6750 — APPROVED** [review]

**Pull request merged**